woocommerce/woocommerce-ios
# UL&S: Remove .showWPComLogin segue

Fixes #2003 
Ref. https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/182

This PR removes the segue `.showWPComLogin` and navigates using the navController.

### To Test
1. Check out this branch
2. `rake dependencies`
3. Build and run

#### Test Case 1 - WP.com email + password
1. Log In > enter WordPress.com email > Next > Enter your password instead.
2. **Expected:** the text link navigates you to the password screen, `LoginWPComViewController`. 

#### Test Case 2 - WP.com email + send magic link + nope, enter password instead
1. Log In > enter WordPress.com email > Next > Send Link > Enter your password instead.
2. **Expected:** the text link navigates you to the password screen.

### Do not merge until
1. The Authenticator PR has been merged. Ref: wordpress-mobile/WordPressAuthenticator-iOS#272
2. A new Authenticator release has been created
3. The release is published on Cocoapods trunk
4. This PR points to the new release

Update release notes:
